spec/features/admin/01_admin_deletes_lifehack_spec.rb

```diff
@@ -0,0 +1,77 @@
+require 'rails_helper'
+
+feature 'admin deletes item', %{
+  As an Admin
+  I want to delete lifehacks and reviews
+  So I can curate the site's content
+} do
+
+  # Acceptance Criteria:
+  # [x] admin can delete lifehack
+  # [x] admin can delete review
+  # [x] member cannot delete lifehack where they are not the creator
+  # [x] member cannot delete review where they are not the creator
+
+  let(:lifehack) { FactoryGirl.create(:lifehack) }
+  let(:lifehack2) { FactoryGirl.create(:lifehack) }
+  let(:user) { FactoryGirl.create(:user, role: "member") }
+  let(:member) { FactoryGirl.create(:user, role: "member") }
+  let(:admin) { FactoryGirl.create(:user, role: "admin") }
+  let(:review) { FactoryGirl.create(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.